base/Dockerfile

```diff
@@ -33,6 +34,16 @@ RUN --mount=type=cache,target=/pkg-cache \
 WORKDIR /work
 
 FROM ${TEST_BASE_IMAGE} AS test-base-rhel
+RUN <<EOT
+#!/usr/bin/env bash
+if ! yum install -y epel-release; then
+  if . /etc/os-release 2>/dev/null; then
+    if [ "$ID" != "fedora" ]; then
+      yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${VERSION:0:1}.noarch.rpm
+    fi
+  fi
+fi
+EOT
```

Comment:
This instruction has been added to be able to install `epel-release` (extra packages) to make sure `bats` is available.